utils/init-common.sh

```diff
@@ -4,15 +4,23 @@ source ${0%/*}/common.sh
 
 message 'Creating qtrpi directory structure'
 
-# sanity check for mounted image
-sudo umount /mnt/raspbian
-sudo umount $ROOT/raspbian/sysroot-full/sys
-sudo umount $ROOT/raspbian/sysroot-full/dev
-sudo umount $ROOT/raspbian/sysroot-full/proc
+if [[ ! $QTRPI_DOCKER ]]; then
```

Comment:
The variable `QTRPI_DOCKER` should be defined in [common.sh](https://github.com/neuronalmotion/qtrpi/blob/5166cc8652b3d2d1de1d308e2ecb93ec03b3894d/utils/common.sh#L74). This is used to centralized all the environment variables in one place.

Reply:
ok, I changed that...